scripts/rebuild_languages.py

```diff
@@ -33,104 +56,103 @@ def downloadSheet(url):
-)
+def main():
+    parser = argparse.ArgumentParser(
+        description="Script to rebuild the language files from the Google Sheet"
+    )
+    parser.add_argument(
+        "--include-wip",
+        action="store_true",
+        help="Include work-in-progress strings, as marked with 'WIP' in the the first column.",
+    )
+    args = parser.parse_args()
+    includeWorkInProgress = args.include_wip
+    languageSpreadsheetURL = (
+        "https://docs.google.com/"
+        "spreadsheets/d/1woTU8dZCHJh7yvdk-N1kgQBUj4Sn3SdRsbKgn6ltJQs/"
+    )
 
-rows = downloadSheet(languageSpreadsheetURL + "export?format=csv")
+    rows = downloadSheet(languageSpreadsheetURL + "export?format=csv")
 
-numHeaders = 5
-headers = rows[:numHeaders]
-assert headers[0][0] == "Go to Documentation", headers[0][0]
-assert headers[1][2] == "English", headers[1][2]
-assert headers[2][2] == "English", headers[2][2]
-assert headers[3][2] == "en", headers[3][2]
+    numHeaders = 5
+    headers = rows[:numHeaders]
+    assert headers[0][0] == "Go to Documentation", headers[0][0]
+    assert headers[1][2] == "English", headers[1][2]
+    assert headers[2][2] == "English", headers[2][2]
+    assert headers[3][2] == "en", headers[3][2]
 
-rows = rows[numHeaders:]
+    rows = rows[numHeaders:]
 
-startColumn = 2
+    startColumn = 2
 
-languages = []
-languageStrings = {}
+    languages = []
+    languageStrings = {}
 
-for columnIndex in range(startColumn, len(headers[1])):
-    languageInEnglish = headers[1][columnIndex]
-    languageInLanguage = headers[2][columnIndex]
-    languageCode = headers[3][columnIndex]
-    languageStatus = headers[4][columnIndex]
-    assert languageCode
-    if not languageStatus.strip():
-        continue
+    for columnIndex in range(startColumn, len(headers[1])):
+        languageInEnglish = headers[1][columnIndex]
+        languageInLanguage = headers[2][columnIndex]
+        languageCode = headers[3][columnIndex]
+        languageStatus = headers[4][columnIndex]
+        assert languageCode
+        if not languageStatus.strip():
+            continue
 
-    languages.append(
-        dict(
-            code=languageCode,
-            langEn=languageInEnglish,
-            langLang=languageInLanguage,
-            status=languageStatus,
+        languages.append(
+            dict(
+                code=languageCode,
+                langEn=languageInEnglish,
+                langLang=languageInLanguage,
+                status=languageStatus,
+            )
         )
-    )
-
-    languageStrings[languageCode] = strings = {}
 
-    for row in rows:
-        key = row[1]
-        if not key.strip():
-            continue
-        string = row[columnIndex]
-        if not string or string == "-":
-            string = languageStrings["en"].get(key, "!missing!")
-        strings[key] = string
+        languageStrings[languageCode] = strings = {}
 
+        for row in rows:
+            if not includeWorkInProgress and "WIP" in row[0]:
```

Comment:
I misread this the first time around. Still I'd like to rephrase for clarity (at least this way it reads more naturally to me):
```suggestion
            if "WIP" in row[0] and not includeWorkInProgress:
```